x-pack/plugins/ml/server/routes/anomaly_detectors.ts

```diff
@@ -6,17 +6,17 @@
 
 import { schema } from '@kbn/config-schema';
 import { wrapError } from '../client/error_wrapper';
-import { licensePreRoutingFactory } from '../new_platform/licence_check_pre_routing_factory';
-import { RouteInitialization } from '../new_platform/plugin';
+import { licensePreRoutingFactory } from './license_check_pre_routing_factory';
+// import { RouteInitialization } from '../new_platform/plugin';
```

Comment:
Are all these commented out lines for the `RouteInitialization` needed in all the routes files?

Reply:
`RouteInitialization` could be reused for my requested change.
https://github.com/elastic/kibana/pull/58680#discussion_r385004425

Reply:
Done in 609be5c4c218d1feed4b249158c438ce7c30f99d